tools/pythonpkg/tests/sqlite/test_types.py

```diff
@@ -81,6 +81,28 @@ def test_CheckInf(self):
         with self.assertRaises(RuntimeError) as context:
             self.cur.execute("insert into test(f) values (?)", (decimal.Decimal('inf'),))
 
+    def test_CheckBytesBlob(self):
+        val = b"Guglhupf"
+        self.cur.execute("insert into test(b) values (?)", (val,))
+        self.cur.execute("select b from test")
+        row = self.cur.fetchone()
+        self.assertEqual(row[0], val)
+
+    def test_CheckMemoryviewBlob(self):
+        sample = b"Guglhupf"
+        val = memoryview(sample)
+        self.cur.execute("insert into test(b) values (?)", (val,))
+        self.cur.execute("select b from test")
+        row = self.cur.fetchone()
+        self.assertEqual(row[0], sample)
+
+    def test_CheckMemoryviewFromhexBlob(self):
+        sample = bytes.fromhex('00FF0F2E3D4C5B6A798800FF00')
+        val = memoryview(sample)
+        self.cur.execute("insert into test(b) values (?)", (val,))
```

Comment:
Could you add a test that passes `None` as a template parameter to a blob?

Reply:
I need to make this test dependent on Python 3, since Python 2 does not have `fromhex()`.

Reply:
Sure, that's fine.